### DISCUSSION TITLE
Student benefits application has been rejected

### DISCUSSION THREAD

**Discussion by DaoSYi** [community/community#48227] [GitHub Education]
I am a student, and I also provided the online proof of school registration from the Chinese Ministry of Education, why the student benefits still can't be certified

**Comment by samus-aran** [comment]
Hi @AlphaSRE  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Comment by whitewatercn** [comment]
可以参考我的这个帖子再逝一逝 https://xlog.whitewater.wang/Github_Student_Developer_Pack
You can refer to this post to try again https://xlog.whitewater.wang/Github_Student_Developer_Pack
